lldb/source/Plugins/LanguageRuntime/Swift/SwiftLanguageRuntimeNames.cpp

```diff
@@ -419,6 +429,66 @@ CreateRunThroughTaskSwitchingTrampolines(Thread &thread,
   return nullptr;
 }
 
+// Search all modules for `target_func` and creates a RunToAddress plan if a
+// single function is found.
+static ThreadPlanSP CreateRunToAddressPlan(StringRef target_func,
+                                           Thread &thread, bool stop_others) {
+  ModuleList modules = thread.GetProcess()->GetTarget().GetImages();
+  SymbolContextList sc_list;
+  modules.FindFunctionSymbols(ConstString(target_func), eFunctionNameTypeFull,
+                              sc_list);
+  if (sc_list.GetSize() != 1 || sc_list[0].symbol == nullptr)
```

Comment:
Why do we care that there's only one?  For instance, if this is an ObjC method, two shared libraries can have implementations of the same ObjC class.  The runtime will pick which one to use, but we don't actually know from symbols which one that is.
And since this is a thread specific breakpoint, the only way setting a breakpoint on the "wrong" function as well as the "right" would cause trouble is if running from the thunk to the target function called the wrong function, which seems unlikely.